swaglyrics/cli.py

```diff
@@ -65,6 +68,8 @@ def get_lyrics(song, artist):
 	:return: song lyrics or None if lyrics unavailable
 	"""
 	url_data = stripper(song, artist)  # generate url path using stripper()
+	if url_data is None:  # url path returned none, pass it along
```

Comment:
This would be a super specific implementation since there would be only one case where stripper returns None and it just passed it along. There is a better solution here.